shell/platform/embedder/embedder.h

```diff
@@ -2434,6 +2470,9 @@ typedef struct {
   /// being registered on the framework side. The callback is invoked from
   /// a task posted to the platform thread.
   FlutterChannelUpdateCallback channel_update_callback;
+
+  /// Data used to initialize Shorebird as part of engine initialization.
+  ShorebirdFlutterProjectArgs shorebird_args;
```

Comment:
Won't this break all non-mac targets?  If you make this a * then it can be optional.

Reply:
I'm not sure that it *matters* that we break non-mac, embedder-api-using targets. 🤷

Reply:
If FlutterProjectArgs is allocated with zeros the this should work fine.

Reply:
This will almost certainly not work with non-mac embedder targets, but I think they'll have problems before this, seeing as we don't support any of them.